### ISSUE TITLE
[Java] The JUnit 5 test method name '...' doesn't match '[a-z][a-zA-Z0-9]*'

### ISSUE THREAD

**Issue by MatteoGioioso** [pmd/pmd#4409]
**Affects PMD Version:** 

**Rule:**

[link]
(https://pmd.sourceforge.io/pmd-6.48.0/pmd_rules_java_codestyle.html#methodnamingconventions)

**Description:**
All my tests are named like that, but this one fails

**Code Sample demonstrating the issue:**

```java
     @Test
        void givenSomething_whenSomethingElse_thenSomething() {
```

**Expected outcome:**

PMD reports a violation at line ..., but that's wrong. That's a false positive.

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
